examples/cognito_forgot_password.py

```diff
@@ -0,0 +1,35 @@
+import asyncio
+
+from aiobotocore.session import AioSession
```

Comment:
We recommend `from aiobotocore.session import get_session` and `session = get_session()` over manual session creation.